src/layers/core/grid-layer/grid-layer-vertex.glsl

```diff
@@ -43,41 +29,42 @@ varying vec4 vColor;
 
 void main(void) {
 
-
   // cube gemoetry vertics are between -1 to 1, scale and transform it to between 0, 1
-  vec2 ptPosition = instancePositions.xy + vec2((positions.x + 1.0 ) * lonOffset / 2., (positions.y + 1.) * latOffset / 2.);
+  vec2 ptPosition = instancePositions.xy + vec2((positions.x + 1.0 ) * lonOffset / 2.0, (positions.y + 1.0) * latOffset / 2.0);
 
   vec2 pos = project_position(ptPosition);
 
-  // if 3d not enabled set elevation to 0
-  float elevation =  mix(0., project_scale(instancePositions.w  * (positions.z + 1.) * ELEVATION_SCALE), extruded);
```

Comment:
I changed all mix() function to if() for better code readability. I don't think there will be any difference in performance... I'll do the perf test later using my latest perf query and see.